src/Facebook/FacebookBatchRequest.php

```diff
@@ -62,16 +62,18 @@ public function __construct(FacebookApp $app = null, array $requests = [], $acce
     }
 
     /**
-     * A a new request to the array.
+     * Adds a new request to the array.
      *
      * @param FacebookRequest|array $request
-     * @param string|null           $name
+     * @param string|null|array     $optionsOrName When it is a string it is the name of the request
```

Comment:
I'd name it just `$options` and describe it like "array of batch request options (e.g. ...). If a string is given, it will be the value of the `name` option." or something like that :)